manage-data/migrate.md

```diff
@@ -107,86 +115,95 @@ Through the {{es}} reindex API, you can connect your new {{es}} Service deployme
+The following steps assume you already have a snapshot repository configured in the old cluster with at least one valid snapshot containing the data you want to migrate.
 
-    ```sh
-    GET /_snapshot
-    GET /_snapshot/_all
-    ```
+### Step 1: Set up the repository in the new cluster [migrate-repo-setup]
 
-2. Get the snapshot name:
+In this step, you’ll configure a snapshot repository in the new cluster that points to the storage location used by the old cluster. This allows the new cluster to access and restore snapshots created in the original environment.
 
-    ```sh
-    GET /_snapshot/NEW-REPOSITORY-NAME/_all
-    ```
+::::{tip}
+If your new {{ech}} or {{ece}} deployment cannot connect to the same repository used by your self-managed cluster, for example if it's a private NFS share, consider one of these alternatives:
 
-    The output for each entry provides a `"snapshot":` value which is the snapshot name.
+* [Back up your repository](/deploy-manage/tools/snapshot-and-restore/self-managed.md#snapshots-repository-backup) to a supported storage system such as AWS S3, Google Cloud Storage, or Azure Blob Storage, and then configure your new cluster to use that location for the data migration.
+* Expose the repository contents over `ftp`, `http`, or `https`, and use a [read-only URL repository](/deploy-manage/tools/snapshot-and-restore/read-only-url-repository.md) type in your new deployment to access the snapshots.
+::::
 
-    ```json
-    {
-      "snapshots": [
-        {
-          "snapshot": "scheduled-1527616008-instance-0000000004",
-          ...
-        },
-        ...
-      ]
-    }
+1. On your old {{es}} cluster, retrieve the snapshot repository configuration:
+
+    ```sh
+    GET /_snapshot/_all
     ```
 
+    Take note of the repository name and type (for example, `s3`, `gcs`, or `azure`), its base path, and any additional settings. Authentication credentials are often stored in the [secure settings](/deploy-manage/security/secure-settings.md) on each node. You’ll need to replicate all this configuration when registering the repository in the new ECH or ECE deployment.
+
+    If your old cluster has multiple repositories configured, identify the repository with the snapshots containing the data that you want to migrate.
 
-3. Add the snapshot repository:
+2. Add the snapshot repository on the new cluster:
 
-    ::::{tab-set}
+    The new cluster must register a snapshot repository that points to the same physical storage location used by the old cluster. This ensures the new cluster can access the existing snapshots.
 
-    :::{tab-item} {{ech}}
+    Considerations:
 
-    From the [console](https://cloud.elastic.co?page=docs&placement=docs-body) of the **new** {{es}} cluster, add the snapshot repository.
+      * If you’re migrating [searchable snapshots](/deploy-manage/tools/snapshot-and-restore/searchable-snapshots.md), the repository name must be identical in the source and destination clusters.
```

Comment:
is this true for a simple restore, or only if you want to restore search functionality?

Reply:
true for a simple restore. if the index being restored is actually a searchable snapshot, the only way to make it working in the destination is as a searchable snapshot where the snapshot is located in a repo with the same name, that's the reason for the message.

As far as I know (but I could be mistaken), there's no way to restore the data from a searchable snapshot index towards a standard index, which by the day would be a great feature and mechanism to convert a searchable snapshot index into a regular index, something I think we don't have today.